src/clr-angular/forms/combobox/option.ts

```diff
@@ -38,36 +63,53 @@ export class ClrOption<T> implements OnDestroy {
   }
 
   private initializeSubscription(): void {
-    this.subscription = this.optionSelectionService.valueChanged.subscribe((value: T) => {
-      // Check for null and undefined needed because if the user doesnt assign a value to the option,
-      // all options should not be selected as the value would be null or undefined
-      if (value === null || value === undefined) {
-        this.selected = false;
-      } else if (this.value === value) {
-        // TODO: Render option when current selection is set by the user
-        this.selected = true;
-      } else {
-        this.selected = false;
-      }
-    });
+    this.subs.push(
```

Comment:
nit: I don't think we have a coding guidline for this but generally I put these subscription pushes into the constructor. It looks like it was laready like this. If you agree consider moving the subscription and array push into the constructor and removing this method.

Reply:
I inherited this code. It's quite common for most of the subscriptions I inherited. We'd better talk if we want to make a convention for this, or leave it to the developer. I guess this was done to keep the code tidy and the constructor small. I personally don't mind either way.